stubs/CoreGenericIterators.phpstub

```diff
@@ -875,11 +901,11 @@ class RecursiveRegexIterator extends RegexIterator implements RecursiveIterator
      * @param RecursiveIterator<TKey, TValue> $iterator
      * @param string $regex
      * @param self::MATH|self::GET_MATCH|self::ALL_MATCHES|self::SPLIT|self::REPLACE $mode
-     * @param int-mask-of<self::USE_KEY> $flags
+     * @param int-mask-of<self::*> $flags
```

Comment:
The only accepted flag is `USE_KEY`: https://www.php.net/manual/en/regexiterator.setflags.php
If Psalm chokes on single-valued bitmask I'd change it to `0|self::USE_KEY`.

Reply:
It's doesn't work either. If only 1 value is accepted, I'll just use the literal. It should work the same.